extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisUtils.scala

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.spark.streaming.kinesis
 
+import com.amazonaws.auth.AWSCredentials
```

Comment:
Updated code a bit after your comments. 

Created ```Credentials``` class to avoid third-party import in api.
Moved credentials parameter to existing constructor as an optional parameter.